hack/docker-builder/Dockerfile

```diff
@@ -8,11 +8,19 @@ RUN curl --output /etc/yum.repos.d/fedora-virt-preview.repo \
 RUN dnf -y install libvirt-devel-${LIBVIRT_VERSION} make git mercurial sudo gcc findutils gradle rsync-daemon rsync qemu-img protobuf-compiler && \
     dnf -y clean all
 
-ENV GIMME_GO_VERSION=1.10
+ENV GIMME_GO_VERSION=1.10.3
 
 RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
 
 ENV GOPATH="/go" GOBIN="/usr/bin"
+ENV KUBEVIRT_VERSION=${KUBEVIRT_VERSION}
+ENV KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER}
+ENV DOCKER_PREFIX=${DOCKER_PREFIX}
+ENV DOCKER_TAG=${DOCKER_TAG}
+ENV IMAGE_PULL_POLICY=${IMAGE_PULL_POLICY}
+ENV TRAVIS_JOB_ID=${TRAVIS_JOB_ID}
+ENV TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST}
+ENV TRAVIS_BRANCH=${TRAVIS_BRANCH}
```

Comment:
is there a specific reason why you added those? For documentation?

Reply:
Oh, now I get it. Might be a good idea to do it this way instead of using the Makefile for passing it.

Reply:
Is this something which requires a specific docker version? The environment variables do not seem to be passed through. For instance the `coveralls` upload on travis fails because it can't see TRAVIS_JOB_ID, ...

Reply:
@rmohr, sorry, this doesn't work.  Premature push on my part.  There doesn't seem to be a way to define container environment variables at build time in Docker, only some kludgy argument passing or at run time via the '--env' parameter.  Consider this broken while I find a better way.  Thanks for the attention :)

Reply:
Don't worry. I think that if you would remove the part which tries to pass environment variables through and make sure that you pass the new environment variable as usual via the Makefile, we could bring this PR in and defer the environment variable discussion to #1352.